index.html

```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Meal Finder App</title>
+    <link rel="stylesheet" type="text/css" href="css/normalize.css" />
+    <link rel="stylesheet" type="text/css" href="css/style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <script src="https://cdn.linearicons.com/free/1.0.0/svgembedder.min.js"></script>
+    <script type="text/javascript" src="script.js"></script>
+    <script src="https://cdn.linearicons.com/free/1.0.0/svgembedder.min.js"></script>
+  </head>
+  <body>
+    <!-- Top Navigation Menu starts -->
+    <nav class="topnav">
+      <div class="active">
+        <a href="#"
+          >Mealoftheday.com
+          <svg class="lnr lnr-dinner">
+            <use xlink:href="#lnr-dinner"></use>
+          </svg>
+        </a>
+      </div>
+      <div id="myLinks">
+        <a href="about.html">About us</a>
+        <a href="#contact">Contact us</a>
+        <a href="#about">Skills</a>
+      </div>
+      <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+        <svg class="lnr lnr-menu">
+          <use xlink:href="#lnr-menu"></use>
+        </svg>
+      </a>
+    </nav>
+    <!-- Top Navigation Menu ends -->
+    <!-- Banner section starts -->
+    <div class="banner">
+      <div class="banner-text">
+        <h1>Hungry? No Cooking Ideas?</h1>
+        <h2>We Are Here To Help You!</h2>
+      </div>
+      <div class="search">
+        <form action="/action_page.php">
```

Comment:
`action="/action_page.php"` do you use it right now?

Reply:
True. I removed it for now.